db/migrate/20170331041221_create_ctas.rb

```diff
@@ -0,0 +1,18 @@
+class CreateCtas < ActiveRecord::Migration[5.0]
+  def change
+    create_table :ctas, id: :uuid do |t|
+      t.string :title
+      t.text :description
+      t.string :website
+      t.boolean :free, default: true
+      t.datetime :start_at
+      t.datetime :end_at
+      t.integer :action_type
+      t.uuid    :location_id
+      t.uuid    :contact_id
+      t.uuid    :call_script_id
+
+      t.timestamps
```

Comment:
nice! Yeah, that 64 character index limit is a rough one. The only way I've seen to push that is to recompile postgres, which is about as close to the bottom of my "What I want to do as a web developer" list as is possible.

FWIW though you can pass the `name: <value>` keword argument to the `add_index` method in a migration to specify your own index name.

At one point in a past job I tried indexing the `reg_advisor_alert_specific_compliance_responsibilites_assignment_id` column. You probably get where that story is going...